.github/workflows/ci.yml

```diff
@@ -18,11 +18,11 @@ jobs:
       matrix:
         include:
           - pair:
-              elixir: "1.11"
-              otp: "21.3"
+              elixir: 1.11.x
+              otp: 21.x
```

Comment:
Why change this to `x`?

Reply:
It supposed to pull the latest version without us explicitly set it.

```
Run erlef/setup-beam@v1
Installing Erlang/OTP OTP-21.3.8.24 - built on amd64/ubuntu-20.04
Using Elixir 1.11.4 (built for Erlang/OTP 21)
Installing Elixir v1.11.4-otp-21
Running mix local.rebar,--force
Running mix local.hex,--force
```

Reply:
Doesn't `1.11` do the same as `1.11.x`?

Reply:
Indeed. The only different is the representation in the job name:

![image](https://github.com/user-attachments/assets/7d0cd33b-dc6f-42cd-b752-e7c3e998ada7)

![image](https://github.com/user-attachments/assets/efbd8916-082b-42f8-b6b7-1e084b65cc85)

Which is your preference?

Reply:
Let's go without the `x` like it was before

Reply:
@whatyouhide Done.

Reply:
@kianmeng removing the quotes is also nasty because now those get treated as integers... Can we just update the latest version, without making any more cosmetic changes? 🙃